hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azurebfs/ITestGetNameSpaceEnabled.java

```diff
@@ -67,10 +69,10 @@ public void testFailedRequestWhenFSNotExist() throws Exception {
             + testUri.substring(testUri.indexOf("@"));
     AzureBlobFileSystem fs = this.getFileSystem(nonExistingFsUrl);
 
-    intercept(AbfsRestOperationException.class,
+    intercept(FileNotFoundException.class,
             "\"The specified filesystem does not exist.\", 404",
             ()-> {
-              fs.getIsNamespaceEnabled();
```

Comment:
getIsNamespaceEnabled() suppresses exceptions and returns corresponding boolean value. Replacing with a FS call so that exact exception is asserted.